src/containers/App/App.js

```diff
@@ -25,12 +26,17 @@ export const App = ({ getEvents }) => {
         </Fragment>
       } />
       <Route path='/profile' component={Profile} />
+      <Route path='/event/:id' render={({ match }) => {
+        const { id } = match.params;
+        return <EventPopUp id={id} />
+      }}
+      />
```

Comment:
I believe that if we were component instead of render, it will implicitly pass the match property on props! If we do that instead and then deconstruct the id in the EventPopUp that would shorten this up.

Reply:
Awesome, I had no idea that was a thing! I will implement it.